src/ResultPaginator.php

```diff
@@ -101,25 +86,6 @@ public function getNextToken()
     }
 
     /**
-     * Fetch the next result for the command managed by the paginator. You may
-     * pass additional arguments, the most appropriate being a limiter.
-     *
-     * @param array $args
-     * @return Result|null
-     */
-    public function getNext(array $args = [])
```

Comment:
I removed this as well. Modifying the paginator options after you begin paginating seems potentially problematic. Maybe we could add a setLimit() function if necessary as that seems to be the only option someone would/should change after they begin iterating.